### ISSUE TITLE
Hitting Enter to start search 

### ISSUE THREAD

**Issue by SophiaD-M** [BetaMasaheft/Documentation#1880]
Is it possible to hit the Enter key to start the search? https://betamasaheft.eu/newSearch.html 
Right now, when I type in the search box and hit enter I only creat a new line. The search has to be activated by clicking with the mouse. 

thank you

**Comment by PietroLiuzzo** [comment]
No, it is not possible because the query goes into a text area, not an input field, where a new line must be possible as such.

**Comment by SophiaD-M** [comment]
ok, thank you

**Comment by thea-m** [comment]
this would be really practical, it is quite counterintuitive right now
